cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/gomint/event/GoMintChannelMessageReceiveEvent.java

```diff
@@ -0,0 +1,28 @@
+package de.dytanic.cloudnet.ext.bridge.gomint.event;
+
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+
+public final class GoMintChannelMessageReceiveEvent extends GoMintCloudNetEvent {
+
+    private final String channel, message;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6226234)